share/spack/gitlab/cloud_pipelines/stacks/spack-snake/spack.yaml

```diff
@@ -0,0 +1,44 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: false
+    static_analysis: true
+
+  packages:
+    all:
+      require:
+      - "%gcc"
```

Comment:
I don't think we want this, because lots of the python packages don't use a compiler at all and the virtual providers config guarantees this just as well

Reply:
Hmm that is a good point. I wen't gcc because we haven't had luck building many python packages with intel (and I just copied e4s to start 🙂 ), but.... it is likely overly restrictive.